### DISCUSSION TITLE
群友们的一些建议05.17.2023 6pm.

### DISCUSSION THREAD

**Discussion by AnzOOCH** [IndustrialCraftMC/IndustrialCraft2#84] [【意见建议】Ideas]
1.增加类似格雷电弧炉那样的机器，可以用于回收旧机器为原材料锭或粉。出于平衡性考虑可以有一定损耗，或者需要一定条件(比如必须通入压缩空气或使用热量而非电力驱动)
2.mfsu需要六个兰波顿水晶合成，但只有四倍的容量，是否需要调整一下两者中的其中一者？
3.压缩机塞进能量水晶粉压缩时，如果产物槽有物品未取出，可能shift点击水晶粉会直接扔进供电槽中去，浪费了一个粉。由此引申出来的，shift快速填充物品的功能最好针对供电槽做一下QoL优化。
4.一次性电池太弱鸡。

**Comment by AnzOOCH** [comment]
5.储物箱不能r键整理，可能并不是原生的“箱子”容器的原因，既然开发组希望尽可能使用原生接口，那我认为这个问题似乎可以迎刃而解。
另外，储物箱的存在，直接让潜影盒的地位尴尬了起来。群友建议参考搬箱器mod的做法，让玩家在携带箱子时获得缓慢debuff。出于游戏性考虑，可以让低级的储物箱不获得或只有低等级debuff，高级的储物箱携带时获得高等级缓慢debuff。这个debuff可以在玩家饮用咖啡后避免，或者在玩家穿戴纳米套装及以上时避免(可以理解为碳纤维外骨骼)，为了能与其他mod的盔甲兼容，可以通过检测盔甲值实现。

**Comment by AnzOOCH** [comment]
6.泵不好用，可参考gt泵，提供采矿管道后从机器下方自动伸出，吸取附近一定范围内的液体源头，吸取完毕后伸到下一格，直到用光采矿管道或顶到固体方块，管道可在摧毁机器后自动回收。泵最好也能超频，现如今无法超频的泵面对超频的洗矿机较为无力。
7.建议更新复制机的ui，选择复制的物品最好能做成列表式的，模式存储机同理，最好能够允许玩家删除不用的物品记录。

**Comment by dysslz** [comment]
1.19.2 forge端 加了工业2就报错 进不去存档  industriacraft-2-2.9.21 ex119-1.19.2-forge

**Reply by dysslz** [reply]
fabric端正常进入游戏
